### DISCUSSION TITLE
essay/2024/08/04/weekend-work

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#227] [Comments]
# essay/2024/08/04/weekend-work

신입사원 때. 주말 출근하는 걸 참 좋아했습니다. 토요일, 일요일엔 느즈막이 일어나서 회사로 출근했습니다. 회사 앞 맥도날드에서 햄버거를 하나 사서 들어가는 게 저의 행복이었습니다.

http://localhost:3000/essay/2024/08/04/weekend-work.html

**Comment by synuns** [comment]
헉..!! 제가 지금 그러고 있는데요. 뭔가 카페보다 텅빈 사무실이 훨씬 편한 느낌이라 주말에도 가게되는 거 같아요.
저는 제가 별종이라고 생각해서 저 같은 사람이 없을 줄 알았는데 여기 계셨군요 ㅎㅎㅎ

**Reply by BenjaminKim** [reply]
하하, 반갑습니다!
